wolfcrypt/src/random.c

```diff
@@ -306,46 +333,55 @@ static int Hash_gen(DRBG* drbg, byte* out, word32 outSz, const byte* V)
 
     XMEMCPY(data, V, sizeof(data));
     for (i = 0; i < len; i++) {
-        if (wc_InitSha256(&sha) != 0 ||
-            wc_Sha256Update(&sha, data, sizeof(data)) != 0 ||
-            wc_Sha256Final(&sha, digest) != 0) {
-
-            return DRBG_FAILURE;
-        }
+    #ifdef WOLFSSL_ASYNC_CRYPT
+        ret = wc_InitSha256_ex(&sha, drbg->heap, drbg->devId);
```

Comment:
Sean you are right, the QAT does have a DRBG "cpaCyDrbgGen" API. I agree that would be better to use their DRBG instead of leveraging the SHA256 hardware accel. I put this on the list for after this PR. I also found an API for testing prime "cpaCyPrimeTest" that we might be able to use for speeding up RSA key gen.